### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Tracy-miranja**
i  have added a contact form and linked it with my form spree.

### CODE REVIEW

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @Tracy-miranja, @martinkarugaba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good commit history :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Formspree correctly setup :heavy_check_mark: 
- Linter checks passing :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,170 +1,262 @@
...
+      <section class="icons">
+        <ul class="footer-social-links">
+          <a href="#"><img src="./images/Vector f1.png" /></a>
+          <a href="#"><img src="./images/Vector-17.svg" style="margin-right: 2px;height: 17px;"><img src="./images/Vector f7.png"/></a>
+          <a href="#"><img src="./images/Vector f3.png" /></a>
+          <a href="#"><img src="./images/Vector f4.png" /></a>
+        </ul>
+        <div class="bar"></div>
+      </section>
+    </footer>
```
Comment (Tadesse-Alemayehu) [inline index.html:260]: ##### Impressive changes you made @Tracy-miranja. But There are some minor details we need to address to get your design close to the Figma design.
- [ ] Please make sure to use the right font as defined in the Figma design. In your case please use `Inter` font family.
- [ ] Please make sure your `about` content is aligned to the center as shown in the Figma design.

### Figma design
Screenshot 2022-12-07 203111

### Your design
Screenshot 2022-12-07 203211

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Tracy-miranja 
### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [ ] Good job making most of the changes requested by the previous reviewer. 🙏
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  All linters passed 🥇

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @Tracy-miranja and @martinkarugaba 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Great implementing changes from the first review 💯 
✔️ Nice UI 💯 

## Optional suggestions
- This project is being approved based on the first reviewer's comments but as the second reviewer has said, I strongly recommend changing the fonts for this project to match the one in the Figma template as this will make the UI much better.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**